Hello @MedAliMarz I pull this branch for testing but I am having an issue when building.

I made sure to remove `node_modules` and redownload them, but it did not work.

```
nodes/EditImage.node.ts(969,20): error TS2503: Cannot find namespace 'gm'.
nodes/EditImage.node.ts(1137,18): error TS2503: Cannot find namespace 'gm'.
```

Can you please confirm it is working properly for you? Thanks!

--- a/packages/nodes-base/nodes/EditImage.node.ts
+++ b/packages/nodes-base/nodes/EditImage.node.ts
@@ -966,6 +966,7 @@ export class EditImage implements INodeType {
 
 			const cleanupFunctions: Array<() => void> = [];
 
+			// @ts-ignore
 			let gmInstance: gm.State;
 
 			const requiredOperationParameters: {
@@ -1134,6 +1135,7 @@ export class EditImage implements INodeType {
 					const resizeOption = operationData.resizeOption as string;
 
 					// By default use "maximumArea"
+					// @ts-ignore
 					let option: gm.ResizeOption = '@';
 					if (resizeOption === 'ignoreAspectRatio') {
 						option = '!';

